plugins/arRestApiPlugin/modules/api/actions/informationobjectsUpdateAction.class.php

```diff
@@ -30,7 +30,7 @@ protected function put($request, $payload)
 
     if (QubitInformationObject::ROOT_ID === (int)$this->io->id)
     {
-      throw new QubitApiForbiddenException;
+      throw new QubitApiForbiddenException('Forbidden');
```

Comment:
I think this is a better approach :smile: 
https://github.com/symfony/http-kernel/tree/master/Exception